app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -1888,6 +2175,24 @@ class ExplorationActivityTest {
     }
   }
 
+  private fun markSpotlightSeen(feature: Spotlight.FeatureCase) {
+    val profileId = ProfileId.newBuilder().setInternalId(internalProfileId).build()
+    spotlightStateController.markSpotlightViewed(profileId, feature)
+    testCoroutineDispatchers.runCurrent()
+  }
+
+  private fun markAllSpotlightsSeen() {
+    markSpotlightSeen(Spotlight.FeatureCase.LESSONS_BACK_BUTTON)
+    markSpotlightSeen(Spotlight.FeatureCase.VOICEOVER_PLAY_ICON)
+    markSpotlightSeen(Spotlight.FeatureCase.VOICEOVER_LANGUAGE_ICON)
+  }
+
+  private fun logIntoUserThrice() {
```

Comment:
Perhaps this should be ``logIntoAdminThrice`` per what it's actually doing?

Reply:
done